osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -108,6 +107,9 @@ private void loadBeatmapSets(IEnumerable<BeatmapSetInfo> beatmapSets)
             itemsCache.Invalidate();
             scrollPositionCache.Invalidate();
 
+            // the filter criteria may have changed since the above filter operation.
```

Comment:
I don't see a filter operation anywhere above, in fact, one seems to have been removed...?

Reply:
Ah yep, this comment is no longer valid. The above filter was incorrect in most every case so I removed it.